regulations/uitests/interp_test.py

```diff
@@ -26,6 +26,9 @@ def test_interps(self):
         # should have the appropriate header
         self.assertTrue('OFFICIAL INTERPRETATION TO 2(h)' in interp_dropdown.text)
 
+        # should have the "SHOW" link
+        self.assertTrue('SHOW' in interp_dropdown.text)
```

Comment:
Slightly nicer way to do this in Python: `self.assertIn('SHOW', interp_dropdown.text)`

Reply:
Fixed in 6381fa7eba9dc31ad2da21ab50f01339ecd78574